css-shared-element-transitions/index.bs

```diff
@@ -593,254 +655,307 @@ document's DOM structure.
-	
-		Issue: Define what active animation means here.
-	
-	1. If no [=page-transition pseudo-elements=] has an active animation:
-	
-        1. Set |manager|&apos;s {{[[Phase]]}} internal slot to "idle".
-
-        1. [=Resolve=] |manager|&apos;s{{[[finished]]}} with |finishedP|.
+    To <dfn>update transition DOM</dfn> given a {{SameDocumentTransition}} |transition|:
+
+    1. For each [=page-transition pseudo-elements=] associated with |transition|,
+        check whether there is an active animation associated with this pseudo-element.
+
+        Issue: Define what active animation means here.
+
+    1. If no [=page-transition pseudo-elements=] has an active animation:
+
+        1. Set |transition|'s [=SameDocumentTransition/phase=] to "`finished`".
+
+        1. [=Resolve=] |transition|'s[=SameDocumentTransition/finished promise=].
 
         1. Return.
-	
-	1. [=map/For each=] |tag| -> |CapturedElement| of |manager|&apos;s {{[[TaggedElements]]}}:
-	
-        1. If |CapturedElement| has an "incoming element", run [=capture the image=]
-            on |CapturedElement|&apos;s "incoming element" and update the displayed
+
+    1. [=map/For each=] |tag| -> |capturedElement| of |transition|'s [=SameDocumentTransition/tagged elements=]:
+
+        1. If |capturedElement| has an "incoming element", run [=capture the image=]
+            on |capturedElement|'s "incoming element" and update the displayed
             image for ''::page-transition-incoming-image'' with the tag |tag|.
 
             At the [=user-agent origin=],
-            set |incoming|&apos;s 'object-view-box' property
+            set |incoming|'s 'object-view-box' property
             to a value that when applied to |incoming|,
             will cause the view box to coincide with "incoming element"'s [=border box=]
             in the image.
 
         1. ...
-    
+
             Issue: Also clarify updating the animation based on new bounds/transform to
             get c0 continuity.
 </div>
 
 <div algorithm>
-	To <dfn>compute the interest rectangle</dfn> of an {{Element}} |el|:
-	
-	1. If |el| is the document's root element, the |interest rectangle| is
-		the intersection of the viewport, including the size of renderered
-		scrollbars (if any), with |el|'s ink overflow rectangle.
-	
-	1. If |el|'s <a href="https://drafts.csswg.org/css-overflow/#ink-overflow-region">ink overflow area</a>
-		does not exceed an implementation-defined maximum size, the |interest rectangle|
-		is equal to |el|'s ink overflow rectangle.
-	
-		Issue: Define the algorithm used to clip the snapshot when it exceeds max size.
+    To <dfn lt="computing the interest rectangle|compute the interest rectangle">compute the interest rectangle</dfn> of an {{Element}} |el|, perform the following steps.
+    They return a rectangle.
+
+    1. If |el| is the document's root element,
+        then return a rectangle that is the intersection of the viewport,
```

Comment:
```suggestion
        then return a rectangle that is the intersection of the layout viewport,
```

There is an explaination here: https://developer.mozilla.org/en-US/docs/Glossary/Layout_viewport but I couldn't find the exact spec where this is defined.